OPUNetTransportLayer.cpp

```diff
@@ -945,27 +945,27 @@ int OPUNetTransportLayer::ReadSocket(SOCKET sourceSocket, Packet& packet, sockad
 	}
 
 	// Check the server port for data
-	unsigned long numBytes;
-	int errorCode = ioctlsocket(sourceSocket, FIONREAD, &numBytes);
-	// Check for success
+	unsigned long byteCountUnsigned; //ioctlsocket sets argp to number of bytes read when passed FIONREAD

```

Comment:
I would drop "Unsigned" from `byteCountUnsigned` as that's just repeating the type.